### DISCUSSION TITLE
RTF to PDF

### DISCUSSION THREAD

**Discussion by sdsbruno** [ImageMagick/ImageMagick#5697] [Help]
Hi, good morning everyone!
If possible, I would like a help. 😊

Does anyone know how to turn RTF files into PDF (for free)?
Until I get an alternative, but it has a watermark.

The application is in Java, but I can add a resource depending on it.
Thanks a lot

**Comment by snibgo** [comment]
Open Office can do that.

**Reply by sdsbruno** [reply]
Hi Snidgo,
But if possible, I would like to convert RTF to PDF without the need for other software. 
I want to do this within my software.
Thanks for the answer.

**Comment by snibgo** [comment]
I don't understand "without the need for other software". Do you want to write your own software, in C or whatever language? Then do so.

This is an ImageMagick forum. ImageMagick cannot read RTF files. You may get better help from a more general forum such as stackoverflow.
